flutter v9: update snippets (#13496)
getsentry/sentry-docs: platform-includes/configuration/auto-session-tracking/dart.flutter.mdx

});
```

#### Web

<Alert>

Requires SDK version `9.0.0` or higher.

</Alert>

> On web it is required to use the `SentryNavigatorObserver` since sessions are tied to navigation events.

Reviewer: ```suggestion
On web you're required to use the `SentryNavigatorObserver` since sessions are tied to navigation events.
```
